sql/moz-fx-data-shared-prod/subscription_platform_derived/daily_active_logical_subscriptions_v1/schema.yaml

```diff
@@ -51,6 +51,9 @@ fields:
   - name: country_name
     type: STRING
     mode: NULLABLE
+  - name: country_state_code
```

Comment:
Somewhat awkwardly named because I don't want it to be ambiguous and interpreted as the state of a subscription